### ISSUE TITLE
Time difference between Inject and within time nodes for same solar event

### ISSUE THREAD

**Issue by TheNutHouse84** [rdmtc/node-red-contrib-sun-position#417]
**Describe the bug**
There is a time difference between the inject node and within time node for same solar event. Inject node Sunset Start = 6:08:23pm within node sunset start = 6:07:12PM. Same type of error noticed for civil dawn. See screen shot.

**Screenshots**
![image](https://user-images.githubusercontent.com/46636354/155827053-6e739736-90e7-4dbe-8178-e71e8457cfa9.png)

**Expected behavior**
Time should be the same

**System information (please complete the following information):**
 - Node Version 16.14.0
 - Node-Red Version 2.2.0

**Additional context**

**Comment by Hypnos3** [comment]
Hello,

This is an expected behavior.

The time-inject node always shows the **next timestamp** - the timestamp of the next inject, whereas the within-time node always shows the timestamps for the current day.
In your example if it is 02/26/22 at 8:45PM, the within-time node shows the times for the 26th.

For 26th february 2022 the civil dwan for your location is 6:34:26, where the next civil dawn is on **27th** 6:35:56.

The reason for this is that the within-time node determines the start and end time for the current day and then checks whether the current time is within this period. These determined times are simply output as status. The inject node always determines the next time and outputs this as status.

Is this here a feature request, because to change in such a way that the within-time node after expiration of the time period outputs the next time period?

regards

**Comment by TheNutHouse84** [comment]
No, now that I understand the status I am ok with the current functionality. Thank you for your contribution.
